reddwarf/data_loader.py

```diff
@@ -1,7 +1,9 @@
+from enum import Enum
 import json
 import os
+from typing import Literal
 from fake_useragent import UserAgent
-from datetime import timedelta
+from datetime import datetime, timedelta, timezone
```

Comment:
Forcing a small change just to get tests to run (they mistakenly weren't running on PRs when you first submitted)

```suggestion
from datetime import datetime, timezone, timedelta
```